### PR TITLE
feat: make PSM bootstrap support multiple brands

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
refs: #6142
refs: #6139

## Description

Make PSM bootstrap support multiple brands

This helps with #6142, but might be merged into that or #6139.

### Security Considerations

The PSM facets are available throughout PSM. That seems acceptable.

### Documentation Considerations

None

### Testing Considerations

Existing tests continue to pass. Since some are using PSM bootstrap, we can tell that it successfully stored one PSM and retrieved it. I presume two wouldn't be harder, but haven't written tests.

### CODE REVIEW

File: packages/inter-protocol/src/proposals/startPSM.js [hunk 1]
```diff
@@ -138,11 +139,20 @@ export const startPSM = async (
...
-  psmGovernorCreatorFacet.resolve(governorFacets.creatorFacet);
+  /** @typedef {import('./econ-behaviors.js').PSMFacets} PSMFacets */
+  /** @type {PSMFacets} */
+  const newPsmFacets = {
+    psm: E(governorFacets.creatorFacet).getInstance(),
+    psmGovernor: governorFacets.instance,
+    psmCreatorFacet: E(governorFacets.creatorFacet).getCreatorFacet(),
+    psmAdminFacet: E(governorFacets.creatorFacet).getAdminFacet(),
+    psmGovernorCreatorFacet: governorFacets.creatorFacet,
+  };
```
Comment (dckc) [inline packages/inter-protocol/src/proposals/startPSM.js]: I wonder if we're going to regret storing these promises.

File: packages/inter-protocol/src/proposals/startPSM.js [hunk 2]
```diff
@@ -217,15 +227,21 @@ export const makeAnchorAsset = async (
 };
 harden(makeAnchorAsset);
 
-/** @param {BootstrapSpace & { devices: { vatAdmin: any }, vatPowers: { D: DProxy }, }} powers */
+/** @typedef {import('./econ-behaviors.js').EconomyBootstrapSpace} EconomyBootstrapSpace */
+
+/** @param {BootstrapSpace & EconomyBootstrapSpace & { devices: { vatAdmin: any }, vatPowers: { D: DProxy }, }} powers */
+// /** @param {BootstrapSpace & { devices: { vatAdmin: any }, vatPowers: { D: DProxy }, }} powers */
```
Comment (dckc) [inline packages/inter-protocol/src/proposals/startPSM.js]: Did you mean to keep this commented-out line?

File: packages/inter-protocol/src/proposals/startPSM.js [hunk 3]
```diff
@@ -217,15 +227,21 @@ export const makeAnchorAsset = async (
...
 export const installGovAndPSMContracts = async ({
   vatPowers: { D },
   devices: { vatAdmin },
   consume: { zoe },
+  produce: { psmFacets },
   installation: {
     produce: { contractGovernor, committee, binaryVoteCounter, psm },
   },
 }) => {
+  psmFacets.resolve(makeScalarMapStore());
```
Comment (dckc) [inline packages/inter-protocol/src/proposals/startPSM.js:255]: Perhaps leave a comment for our future selves about what this is doing here?

**Review by dckc — APPROVED** [review]
Looks like good progress toward the target.

File: packages/inter-protocol/src/proposals/startPSM.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/proposals/startPSM.js]: You're probably right.

File: packages/inter-protocol/src/proposals/startPSM.js (hunk 2 above)
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/proposals/startPSM.js]: nope.

File: packages/inter-protocol/src/proposals/startPSM.js (hunk 3 above)
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/proposals/startPSM.js:255]: done

**Pull request merged**